src/FluentAssertions.Analyzers/FluentAssertions.Analyzers.nuspec

```diff
@@ -25,6 +25,7 @@
     <releaseNotes>
       See https://github.com/fluentassertions/fluentassertions.analyzers/releases/
     </releaseNotes>
+    <repository type="git" url="https://github.com/fluentassertions/fluentassertions.analyzers.git">
```

Comment:
I don't know how strict the xml parsing is, but all examples I've encountered uses a self closing tag.